django_telegrambot/apps.py

```diff
@@ -243,12 +251,14 @@ def ready(self):
                     logger.error('Error: "{}"'.format(er.message))
                     return
 
-            DjangoTelegramBot.bots.append(bot)
-            DjangoTelegramBot.bot_tokens.append(token)
-            DjangoTelegramBot.bot_usernames.append(bot.username)
+            bot_data['bot'] = bot
+            DjangoTelegramBot.bots_data[bot_data['token']] = bot_data
 
-
-        logger.debug('Telegram Bot <{}> set as default bot'.format(DjangoTelegramBot.bots[0].username))
+        first_bot = list(DjangoTelegramBot.bots_data.values())[0]
```

Comment:
The username is not available before the webhook setup. Thats why you show id or token instead. Did I got that right?

Reply:
Yes that’s the reason.